test/gtest/ucp/ucp_test.cc

```diff
@@ -97,6 +97,10 @@ void ucp_test::cleanup() {
 void ucp_test::init() {
     test_base::init();
 
+    if (RUNNING_ON_VALGRIND && has_only_transport("ud_v")) {
+        UCS_TEST_SKIP_R("Skip UCP UD verbs tests with valgrind");
```

Comment:
maybe we can do it when we instantiate the test - check for which transports are available?
see https://github.com/openucx/ucx/blob/60497102248805c00e0d61f2d9a7f6389a39fe54/test/gtest/ucp/ucp_test.cc#L566

Reply:
Done.